config/environments/development.rb

```diff
@@ -53,4 +53,6 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
```

Comment:
Does'nt rubocop complain about this extra line?

Reply:
No, rubocop inspects 9 files and says no offenses

Reply:
Ok, then delete the extra line :sweat_smile: please!!! :pray: